slk/repl/repl.py

```diff
@@ -1032,11 +1032,47 @@ def do_new_account(self: SidechainRepl, line: str) -> None:
             return
         args.pop(0)
 
-        for alias in args:
+        # an new account either generates its own new secret seed, or one may be
+        # specified by using a `-s <seed>` switch
+        aliases = []
+        seeds = []
+        index = 0
+        while True:
+            if index == len(args):
+                break
+            if args[index] == '-s':
+                print(
+                    "Error: new_account -s switch must come after the alias argument. Type "
+                    '"help" for help.'
+                )
```

Comment:
There should be a `return` after this, so it doesn't keep processing after this.

Reply:
Fixed in `491c3e8 fixup! feat: Add load_batch command (#89)`

Reply:
Fixed in `491c3e8 fixup! feat: Add load_batch command (#89)`